docs/source/key_components/rl_toolkit.rst

```diff
@@ -113,17 +119,30 @@ In asynchrounous learning, the policy manager is the centerpiece of the policy s
 Individual policy updates, however, may or may not occur within the policy manager itself depending
 on the policy manager type used. The provided policy manager classes include:
 
-* ``LocalPolicyManager``, where the policies are updated within the manager itself;
-* ``MultiProcessPolicyManager``, which distributes policies amongst a set of trainer processes to parallelize
-  policy update;
-* ``MultiNodePolicyManager``, which distributes policies amongst a set of remote compute nodes to parallelize
-  policy update;
+* ``SimplePolicyManager``, where the policies are updated within the manager itself, sequentially or in parallel through multi-processing;
+* ``DistributedPolicyManager``, which distributes policies amongst a set of remote compute nodes to parallelize policy update.
 
+Moreover, in ``data-parallel`` mode, each policy manager has an additional trainer(``grad_worker``)
+allocator, which provides a policy-to-trainer mapping. The trainer allocator performs auto-balance
+during training, by dynamically adjusting trainer number for policies according to the
+experience/agent/policy number.
 
 .. image:: ../images/rl/policy_manager.svg
     :target: ../images/rl/policy_manager.svg
     :alt: RL Overview
 
+The ``DistributedPolicyManager`` runs a set of ``policy_host`` and a ``TrainerAllocator``.
+``policy_host`` is a process/VM/node that hosts the update of a policy. The ``TrainerAllocator``
+dynamically adjusts trainer node numbers for policies according to the experience/agent/policy
+number. Each ``policy_host`` independently updates its own policies for policy-level parallelism. 
+
+During training, the ``PolicyManager`` first receive experience data from ``RolloutManger``,
+then send them to according ``policy_host``. Each ``policy_host`` will send gradient tasks consist
+of policy state and experience batch, to several stateless ``grad_worker`` for gradient computation.
+The ``grad_worker`` or trainer is stateless, which means it will not rely on previous policy states,
```

Comment:
The ``grad_worker`` is stateless and computes gradients using the policy state and data batch supplied in a task.